fastlane/Fastfile

```diff
@@ -241,7 +241,7 @@ platform :android do
     UI.success("Done! New Release Version: #{release_version_current}. New Build Code: #{build_code_current}")
   end
 
-  lane :finalize_hotfix_release do
+  lane :finalize_hotfix_release do |options|
```

Comment:
I added options to finalize_hotfix_release lane to fix this error when finalizing hotfix release:

![image](https://github.com/Automattic/pocket-casts-android/assets/1405144/8fc07c83-e689-46fb-9036-09768b77c725)